web-marketplace/src/components/organisms/DashboardNavigation/DashboardNavigation.tsx

```diff
@@ -46,6 +48,7 @@ export const DashboardNavigation = ({
   collapsed: boolean;
   walletConnect?: boolean;
   hasProjects?: boolean;
+  hasCreditBatches?: boolean;
```

Comment:
you've already added `hasCreditBatches` to `DashboardNavigationProps` so it seems redundant